src/store/contact/contactService.ts

```diff
@@ -0,0 +1,10 @@
+import { apiService } from '@/services/APIService';
+import type { ContactLabelsAPIResponse, ContactLabelsPayload } from './contactTypes';
+
+export class ContactService {
+  static async getContactLabels(payload: ContactLabelsPayload): Promise<ContactLabelsAPIResponse> {
+    const { contactId } = payload;
+    const response = await apiService.get<ContactLabelsAPIResponse>(`contacts/${contactId}/labels`);
+    return response.data;
+  }
```

Comment:
We could just return apiService.get instead data as the data handling is done at the actions.